lib/helpers.bash

```diff
@@ -107,6 +106,10 @@ bash-it ()
         else
             if _is_function ${func}es; then
                 func=${func}es
+            elif [[ $func =~ ^_bash-it-additional- ]]; then
```

Comment:
See above, this would not be necessary if you followed the `case` structure above.